src/components/ResultsList.vue

```diff
@@ -1,23 +1,20 @@
 <template>
   <section class='results-list'>
-    <h1>Result List</h1>
-    <section :key='i' v-for='(result, i) in searchResults'>
+    <h4 v-if="!$store.state.searchResults.length">
```

Comment:
Glad we got this condition in here.  We will have to refactor this so that it shows a loading icon or message until a certain amount of time has passed, then we can show `No results found` with `Find Park`.